### ISSUE TITLE
Suggestion: ARIA list on Carousel

### ISSUE THREAD

**Issue by thejimbirch** [twbs/bootstrap#23721]
There has been discussion, and incorrectly implemented ARIA listbox was removed in issues #22061 and #22543 by @patrickhlauke 

I would like to suggest the ARIA `list` https://www.w3.org/TR/wai-aria/roles#list and `listitem` https://www.w3.org/TR/wai-aria/roles#listitem roles to the `.carousel-inner` and `.carousel-item`.

Pull request to follow.

**Comment by thejimbirch** [comment]
@bardiharborow The Pull Request that referenced this was closed as stale #23722 and not reviewed by the team member that was supposed to.  I assume this issue should be closed also.

**Comment by Glenn-Domin** [comment]
There's a pretty good carousel example at [https://www.w3.org/TR/wai-aria-practices-1.1/examples/carousel/carousel-1.html](https://www.w3.org/TR/wai-aria-practices-1.1/examples/carousel/carousel-1.html). The w3 example seems to use `role="group"` on individual slides while the parent has `aria-live="polite"`. Not sure how this would behave with sliding/fading content through.

**Comment by mdo** [comment]
Closing as stale here as well since the PR was closed some time ago. https://github.com/twbs/bootstrap/pull/23722#issuecomment-463841121

**Comment by patrickhlauke** [comment]
Carousels are tricky beasts, and there's no one true "good" pattern for a general purpose carousel - it's often dependent on the type of content slides contain (just images/captions, or actual fully structured/interactive content, etc). I may revisit this hairy topic, but yes for now I'd hold off going down a particular route.
